contracts/Wrapped1155Factory.sol

```diff
@@ -170,16 +196,22 @@ contract Wrapped1155Factory is ERC1155Receiver {
         );
```

Comment:
Do we have a source for the minimal proxy code above? it might be based on https://medium.com/coinmonks/the-more-minimal-proxy-5756ae08ee48 I would also mention this somewhere (or at least copy the details of the execution tree into this repo and link to the eip this was based on: https://eips.ethereum.org/EIPS/eip-1167)

Reply:
Thank you, yes that's the reference for the More-minimal proxy :). I will include the references it into the [Wiki](https://github.com/gnosis/1155-to-20/wiki) and as a comment on the contract